CribblyBackend/Services/TournamentService.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using Dapper;
+
+namespace CribblyBackend.Services
+{
+    public interface ITournamentService
+    {
+        Task<Tournament> GetNextTournament();
+        Task<Tournament> Create(DateTime date);
+    }
+    public class TournamentService : ITournamentService
+    {
+        private readonly IDbConnection connection;
+        public TournamentService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+        public async Task<Tournament> Create(DateTime date)
+        {
+            await connection.ExecuteAsync(
+                @"INSERT INTO Tournaments (Date) VALUES (@Date)",
+                new { Date = date }
+            );
+            return (await connection.QueryAsync<Tournament>(
+                @"SELECT * FROM Tournaments WHERE Id = LAST_INSERT_ID()"
+            )).First();
+        }
+
+        public async Task<Tournament> GetNextTournament()
```

Comment:
One thought on solving the "past midnight" problem: tournaments could have an additional two fields: `IsOpenForRegistration` and `IsActive`. We'd then return the tournament in the DB that's open for registration instead of finding based on date. We could also make the landing page say that a tournament is currently happening if there's a tournament in the DB with `IsActive` set to true. Thoughts?

Reply:
That's a great idea. As long as we ensure that any admin controls we add will properly "enable" and "disable" tournaments properly, that should work! We could use `IsActive` and  `IsOpenForRegistration` as ways of controlling when people will have the option to create teams as well.

Reply:
Done.